src/Platform.Handlers/src/Xamarin.Platform.Handlers/Handlers/View/AbstractViewHandler.cs

```diff
@@ -72,18 +84,43 @@ public virtual void SetView(IView view)
 				}
 			}
 
+			if (VirtualView != null)
+			{
+				SetUpNewVirtualView(VirtualView);
```

Comment:
Does it make sense that this is called before `UpdateProperties` ?  I think so

Reply:
I think so.